argopy/tests/test_xarray.py

```diff
@@ -51,3 +51,17 @@ def test_point2profile2point():
         assert ds_pts.argo.point2profile().argo.profile2point().equals(ds_pts)
     except ErddapServerError:  # Test is passed when something goes wrong because of the erddap server, not our fault !
         pass
+
+
```

Comment:
Can you add testing for all possible errors as well ?
So that the full new function is covered.

This would include 
- the "RuntimeError" and "ValueError" from the utilities.linear_interpolation_remap function (tests to go in test_utilities.py)
- the "ValueError" from the xarray accessor when no profiles can be interpolated (test to go in test_xarray.py)